packages/jellyfish-crypto/src/aes256.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Implementation reference: https://github.com/JamesMGreene/node-aes256
+ */
+import createHash from 'create-hash'
+import randomBytes from 'randombytes'
+import aes from 'browserify-aes'
+
+const CIPHER_ALGORITHM = 'aes-256-ctr'
+
+/**
+ * Encrypt a clear-text message using AES-256 plus a random Initialization Vector.
+ * @param {Buffer} key  A passphrase of any length to used to generate a symmetric session key.
+ * @param {Buffer} data  The clear-text message or buffer to be encrypted.
+ * @param {() => Buffer} initVector Initialization vector provider, then to create AES cipher, default using `crypto` or browserify `random-bytes` package
+ * @returns {Buffer}
+ */
+function encrypt (key: Buffer, data: Buffer, initVector?: () => Buffer): Buffer {
+  const sha256 = createHash('sha256')
+  sha256.update(key)
+
+  const iv = initVector === undefined ? randomBytes(16) : initVector()
+
+  if (iv.length !== 16) {
+    throw new Error('Initialization vector must be 16 bytes long')
+  }
+  const cipher = aes.createCipheriv(CIPHER_ALGORITHM, sha256.digest(), iv)
+  const ciphertext = cipher.update(data)
+  return Buffer.concat([iv, ciphertext, cipher.final()])
+}
+
+/**
+ * Decrypt an encrypted message back to clear-text using AES-256 plus a random Initialization Vector.
+ * @param {Buffer} key A passphrase of any length to used to generate a symmetric session key.
+ * @param {Buffer} encrypted The encrypted message to be decrypted.
+ * @returns {Buffer} The original plain-text message or buffer.
+ */
+function decrypt (key: Buffer, encrypted: Buffer): Buffer {
+  if (encrypted.length < 17) {
+    throw new Error('Provided "encrypted" must decrypt to a non-empty string or buffer')
+  }
+
+  const sha256 = createHash('sha256')
+  sha256.update(key)
+
+  // Initialization Vector
+  const iv = encrypted.slice(0, 16)
```

Comment:
```suggestion
  const initVector = encrypted.slice(0, 16)
```

If you comment might be better to write out the full name.